### DISCUSSION TITLE
Colour of the highlights

### DISCUSSION THREAD

**Discussion by dummifiedme** [pymupdf/PyMuPDF#1574] [Looking for help] [answered]
I tried looking for colours of the annotations with `annots.color`, but it seems to give out wrong values. 
For a yellowish highlight, it is providing a value `(1,1,0)` as stroke and `none` as fill. 

How can I use the module correctly?

**Comment by JorjMcKie** [comment, marked as answer]
Text marker annotations only have stroke colors and no fill color!
Even highlight's color is stroke - although an area is being colored.
Go ahead and trust the result - or look at PDF specification 😉

**Reply by dummifiedme** [reply]
My bad.
I was using (255, 255, 0) to compare it with, forgot to naturalize it.

Thanks. I do trust the code :)
